src/bindings/python/src/pyopenvino/CMakeLists.txt

```diff
@@ -114,9 +114,7 @@ if(OpenVINO_SOURCE_DIR OR OpenVINODeveloperPackage_FOUND)
     install(DIRECTORY ${OpenVINOPython_SOURCE_DIR}/src/openvino
             DESTINATION ${OV_CPACK_PYTHONDIR}
             COMPONENT ${OV_CPACK_COMP_PYTHON_OPENVINO}_${pyversion}
-            ${OV_CPACK_COMP_PYTHON_OPENVINO_EXCLUDE_ALL}
-            USE_SOURCE_PERMISSIONS
-            PATTERN "test_utils" EXCLUDE)
+            ${OV_CPACK_COMP_PYTHON_OPENVINO_EXCLUDE_ALL})
```

Comment:
looks like after this change, tests code will be a part of the product. We should avoid it

Reply:
Seems like it was a debug change, I've removed it, [the pipeline is passing](https://github.com/openvinotoolkit/openvino/actions/runs/5505603283).